### DISCUSSION TITLE
How to center the canvas on a node?

### DISCUSSION THREAD

**Discussion by Vadorequest** [reaviz/reaflow#62] [Q&A]
I'd like to center the canvas on a specific node (not necessarily zoom in/out, but re-center the viewport around the target)

Is there an utility for this? Are there examples for this? 

Maybe it'd be built-in and provided through the `canvasRef`?

**Comment by amcdnl** [comment]
Oh interesting - I have done this before for network based graph uis but didn't think about it for this. Theortically its possible but probably not easy. Can you open a ticket for this?

**Comment by himanshu-1034** [comment]
is this functionality supported? now?

**Comment by himanshu-1034** [comment]
@amcdnl I just want to center the flow chart on a node on the mount of component. is there any hack for now which I can implement in this scanrio.

**Comment by brootle** [comment]
this is crazy that it's 2023 and this basic functionality is still missing 🤦‍♂️ infinite panning should be added and when user wants to center on specified node, the whole canvas should be moved in a way to make the node in the center
